cmd/ecfg/actions.go

```diff
@@ -35,15 +38,15 @@ func decryptAction(filePath string, keydir, outFile string, ftype ecfg.FileType)
 		if err != nil {
 			return err
 		}
-		out, err := ecfg.DecryptData(data, keydir, ftype)
+		out, err := ecfg.DecryptData(data, []string{keydir}, ftype)
```

Comment:
yeah, it constructs a string array (slice, really) using that one string `keydir` as the only member.